grazer/src/main/java/edu/asu/diging/grazer/core/domain/IFileImpl.java

```diff
@@ -0,0 +1,35 @@
+package edu.asu.diging.grazer.core.domain;
+
+import java.util.Date;
+import java.util.List;
+
+public interface IFileImpl {
```

Comment:
an interface should not be called XXImpl. "Impl" implies that it is the implementation of an interface.

Reply:
also, needs javadoc.